firefox_metrics/explores/glean_desktop_engagement.explore.lkml

```diff
@@ -0,0 +1,23 @@
+include: "../views/glean_desktop_engagement.view.lkml"
+include: "/shared/views/countries.view.lkml"
+include: "//looker-hub/firefox_okrs/datagroups/glean_desktop_engagement_last_updated.datagroup.lkml"
+
+explore: glean_desktop_engagement {
+  label: "Engagement for Firefox Desktop (Glean-based)"
+  view_name: glean_desktop_engagement
+
+  always_filter: {
+    filters: [
+      desktop_engagement.submission_date: "28 days",
+      desktop_engagement.is_desktop: "Yes"
+    ]
+  }
+
+  join: countries {
+    type: left_outer
+    relationship: one_to_one
+    sql_on: ${desktop_engagement.country} = ${countries.code} ;;
+  }
```

Comment:
The dimension references here appear to be using the wrong view name, which is causing LookML validation to fail and blocking LookML development via the Looker UI.
```suggestion
  always_filter: {
    filters: [
      glean_desktop_engagement.submission_date: "28 days",
      glean_desktop_engagement.is_desktop: "Yes"
    ]
  }

  join: countries {
    type: left_outer
    relationship: one_to_one
    sql_on: ${glean_desktop_engagement.country} = ${countries.code} ;;
  }
```